src/js/components/stock-movement-wizard/combined-shipments/AddItemsPage.jsx

```diff
@@ -350,29 +335,35 @@ class AddItemsPage extends Component {
     }));
   }
 
-  setLineItems(response, startIndex) {
+  setLineItems({
+    response,
+    startIndex,
+    fetchedLineItems,
+    append,
+  }) {
     const { data } = response.data;
     const lineItemsData = _.map(
       data,
-      val => ({
+      (val) => ({
         ...val,
         disabled: true,
         referenceId: val.orderItemId,
       }),
     );
 
-    this.setState({
-      values: {
-        ...this.state.values,
-        lineItems: this.props.isPaginated && !_.isNull(startIndex) ?
-          _.uniqBy(_.concat(this.state.values.lineItems, lineItemsData), 'id') : lineItemsData,
-      },
-    }, () => {
-      if (!_.isNull(startIndex) && this.state.values.lineItems.length !== this.state.totalCount) {
-        this.loadMoreRows({ startIndex: startIndex + this.props.pageSize });
-      }
+    append?.('lineItems', lineItemsData);
+
+    if (data.length < 10) {
       this.props.hideSpinner();
+      return;
+    }
+
+    this.loadMoreRows(append)({
+      startIndex: startIndex + this.props.pageSize,
+      fetchedLineItems: fetchedLineItems + lineItemsData.length,
```

Comment:
at first thought I would think that `fetchedLineItems` actually store the items, not the size of the items, so how about renaming it to `fetchedItemsSize`/`fetchedLineItemsSize`/`fetchedLineItemsCount`?

Reply:
> changed and what are just the lint fixes, so I would not call the fix eslint on a whole file
btw, I think we should run the linter for the entire project some time ago.

Reply:
``` 
fetchedLineItems: fetchedLineItems + lineItemsData.length,
```
I noticed that it's not used anymore, so I am deleting it :smile: